virt/live_migration/virt-enabling-cclm-for-vms.adoc

```diff
@@ -0,0 +1,29 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-enabling-cclm-for-vms"]
+= Enabling cross-cluster live migration for virtual machines
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: virt-enabling-cclm-for-vms
+
+toc::[]
+
+Cross-cluster live migration enables users to move a virtual machine (VM) workload from one {product-title} cluster to another {product-title} cluster without disruption. You enable cross-cluster live migration by setting cluster feature gates in {VirtProductName} and {mtv-first}.
```

Comment:
```suggestion
Cross-cluster live migration enables users to move a virtual machine (VM) workload from one {product-title} cluster to another without disruption. You enable cross-cluster live migration by setting cluster feature gates in {VirtProductName} and {mtv-first}.
```
I don't think the second OCP mention is needed here, the meaning is clear without it.